examples/05_stable_diffusion/scripts/download_pipeline.py

```diff
@@ -14,7 +14,11 @@
 #
 import click
 import torch
+import os
 from diffusers import StableDiffusionPipeline
+from aitemplate.utils.import_path import import_parent
+import_parent(filepath=__file__, level=1)
```

Comment:
Nit, can this be moved into the `__main__`?

Reply:
What is blocking this from being moved into `if __name__ == "__main__"` section?